src/services/routeService.ts

```diff
@@ -15,10 +15,9 @@ export interface IMultipleRoutesQueryResult {
 }
 
 export default class RouteService {
-    // TODO: refactor undefined to null?
-    public static async fetchRoute(routeId: string): Promise<IRoute | undefined> {
+    public static async fetchRoute(routeId: string): Promise<IRoute | null> {
```

Comment:
well, this depends on which advice we follow on this undefined vs null thing..

Reply:
But, I think we this far have used null in similar cases. So, I would suggest using null here as well, just to keep it consistent. Then we might refactor all of those nulls to undefined if we later agree on that.